### ISSUE TITLE
Change package structure

### ISSUE THREAD

**Issue by thiamsantos** [thiamsantos/vanilla-commons#10]
Create a monorepo structure using [lerna](https://github.com/lerna/lerna) and publish the packages under a [scope](https://docs.npmjs.com/misc/scope).

**Comment by renanpvaz** [comment]
How should it be done? Should all folders inside /lib become packages named as vanilla-commons-<type>?

**Comment by thiamsantos** [comment]
@renanpvaz Yes but the packages should be using scopes, something like `@commons/date` or `@vanilla/date`. Do you think that's a good idea?

**Comment by renanpvaz** [comment]
Seems so, do you know any example project using it that way?

**Comment by thiamsantos** [comment]
I know that [angular](https://github.com/angular/angular) and [react-pdf](https://github.com/diegomura/react-pdf) use this approach.
